example-widget-mui/src/DicePage/DicePage.test.tsx

```diff
@@ -64,7 +64,10 @@ describe('<DicePage />', () => {
     await expect(
       screen.findByRole('heading', { name: /dice/i }),
     ).resolves.toBeInTheDocument();
-    expect(await axe(container)).toHaveNoViolations();
+
+    await act(async () => {
```

Comment:
It looks strange to wrap an expect with an act.
For this place here, if I remove the `act` the test still works and there is no error nor a console warning…
→ Remove the `act` here

Can you also check other places similar to this?

Reply:
we will need to review what to do on some of these as removing the `act` does trigger console messages associated with state updates on the React components.